app/services/retry_task_service.rb

```diff
@@ -0,0 +1,51 @@
+class RetryTaskService < TaskService
+  LOST_TIME_INTERVAL = ClowderConfig.instance["LOST_TIME_INTERVAL"].to_i * 60
+
```

Comment:
@hsong-rh We should keep a counter of the number of tasks for which we dispatched a TowTruck so we can let the UI know in the future when it makes a REST API call to recover some of these tasks.

Reply:
Added `towing_tasks` method in `LaunchJobTask` for this purpose